requirements.txt

```diff
@@ -60,5 +60,5 @@ swagger-spec-validator==2.4.3
 urllib3==1.25.7             # via requests
 validators==0.14.0 # pyup: <=0.10.1 # pin until require_tld supported again
 vine==1.3.0               # via amqp
-werkzeug==0.16.0          # via flask
+werkzeug==1.0.1           # via flask
```

Comment:
👍

Reply:
I try to minimize surprise breakage by sneaking a few dependency updates into each release, but we haven't had one recently, so we're playing a bit of catch up here :tomato: